templates/tempest/bin/invoke_tempest

```diff
@@ -4,7 +4,7 @@ set -x
 TEMPEST_DIR=$HOME/openshift
 OPERATOR_ETC=/etc/test_operator
 # NOTE: might be backed with a PV
-OUTPUT_PATH=$HOME/output
+OUTPUT_PATH=${TEMPEST_OUTPUTDIR:-$HOME/output}
```

Comment:
@afazekas do we want to still keep the invoke_tempest script? I think we should stick with only one script for tempest execution. And shouldn't we make the same change in the [run_tempest.sh](https://github.com/openstack-k8s-operators/tcib/blob/main/container-images/tcib/base/os/tempest/run_tempest.sh) in tcib?

Reply:
I'm removing the calling of the invoke_tempest script in [this](https://github.com/openstack-k8s-operators/test-operator/pull/10) PR - maybe I should update the description of the PR. I think we need to agree how we will approach this.